src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -106,6 +107,11 @@ export default function FormNav(props) {
   // https://github.com/department-of-veterans-affairs/va.gov-team/issues/12323
   useEffect(
     () => {
+      // Need to provide shadowRoot for focusing on elements inside shadow-DOM
+      const root = formConfig.v3SegmentedProgressBar
+        ? document.querySelector('va-segmented-progress-bar').shadowRoot
```

Comment:
the other tricky thing to consider, is sometimes the the element will be found, but the shadowRoot may not exist yet. so this type of selector is not necessarily reliable.  this is why querySelectorWithShadowRoot function was created (async)

Reply:
Thx, Robert!  Yeah, I'm making the query async in my next PR.

Reply:
[Refactors pushed](https://github.com/department-of-veterans-affairs/vets-website/pull/27784/commits/1ab1e616f18f181ba0a95b4d8a78a3d99e6046be).  Now focus code's moved out of effect-hook to an async handleFocus method where I'm using your querySelectorWithShadowRoot to first grab the shadow-host before accessing .shadowRoot.